scripts/build-data.js

```diff
@@ -90,29 +94,29 @@ const getLowestImplementedVersion = ({ features }, env) => {
   );
 
   let envTests = tests
-  .map(({ res: test, name, isBuiltIn }, i) => {
-    // Babel itself doesn't implement the feature correctly,
-    // don't count against it
-    // only doing this for built-ins atm
-    if (!test.babel && isBuiltIn) {
-      return "-1";
-    }
+    .map(({ res: test, name, isBuiltIn }, i) => {
+      // Babel itself doesn't implement the feature correctly,
+      // don't count against it
+      // only doing this for built-ins atm
+      if (!test.babel && isBuiltIn) {
+        return "-1";
+      }
 
-    // `equals` in compat-table
-    Object.keys(test).forEach((t) => {
-      test[invertedEqualsEnv[t]] = test[t];
+      // `equals` in compat-table
+      Object.keys(test).forEach((t) => {
+        test[invertedEqualsEnv[t]] = test[t];
+      });
+
+      return Object.keys(test)
+      .filter((t) => t.startsWith(env))
+      // Babel assumes strict mode
+      .filter((test) => tests[i].res[test] === true || tests[i].res[test] === "strict")
+      // normalize some keys
+      .map((test) => envMap[test] || test)
+      .filter((test) => !isNaN(parseInt(test.replace(env, ""))))
+      .shift();
```

Comment:
This block + the one below it are whitespace only changes... I'm not sure why GitHub is showing it like this.